charts/rollout-operator/templates/deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
           - -kubernetes.namespace={{ .Release.Namespace }}
           {{- if .Values.webhooks.enabled }}
           - -server-tls.enabled=true
-          - -server-tls.self-signed-cert.secret-name={{ .Values.webhooks.selfSignedCertSecretName }}
+          - -server-tls.self-signed-cert.secret-name={{ .Values.webhooks.selfSignedCertSecretName | default (include "rollout-operator.fullname" . )  }}
```

Comment:
can `{{ .Values.webhooks.selfSignedCertSecretName | default (include "rollout-operator.fullname" . )  }}` maybe go in a named template in _helpers.tpl? that way we don't have to remember to update it in multiple places next time